packages/threat-composer/src/components/generic/GenericEntityCreationCard/index.tsx

```diff
@@ -25,6 +25,11 @@ import Textarea, { TextAreaProps } from '../../generic/Textarea';
 import MetadataEditor from '../EntityMetadataEditor';
 import GenericCard from '../GenericCard';
 
+export interface GenericEntityCreationCardRef {
```

Comment:
Suggest to rename to GenericEntityCreationCardRefProps to be consistent with other naming conventions.